protos/sources/vega/governance.proto

```diff
@@ -66,12 +78,52 @@ message NewMarketConfiguration {
   string quadratic_slippage_factor = 10;
 }
 
+// Configuration of a new successor market
+message SuccessorMarketConfiguration {
+  // The ID of the market this proposal should succeed
```

Comment:
```suggestion
  // The ID of the market that the proposed market should succeed.
```

Reply:
Kind of on the fence on this one. A successor proposal isn't a full market proposal. This reads more like a successor proposal is a market proposal that has a parent ID tacked on to it, whereas it's more of a set of changes needed to create a new market that takes over from the parent market. @candida-d

Reply:
@wwestgarth @EVODelavega How about: 
  // The ID of the market that the successor replaces 
 ?
 
 Unsure about the word replaces - I hope someone has a better idea.